.github/workflows/ci-unit-tests.yml

```diff
@@ -26,9 +26,10 @@ jobs:
 
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
 
-    - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe
+    - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
```

Comment:
sometimes dependabot adds version at the end, but not everywhere. I wonder if adding it manually once will cause it to keep adding it.